src/geovista/bridge.py

```diff
@@ -717,10 +719,10 @@ def from_points(
     @classmethod
     def from_tiff(
         cls,
-        fname: PathLike,
+        fname: Path | str,
         name: str | None = None,
-        band: int | None = 1,
```

Comment:
MyPy couldn't cope with this potentially being `None`, and as far as I could tell it would be impossible for it to be `None` anyway since it has a default?

Frankly I would have liked to do the same for `rgb` as well but this triggers [Ruff `FBT001`](https://docs.astral.sh/ruff/rules/boolean-type-hint-positional-argument/), which seems fair enough.